cookbooks/aws-parallelcluster-config/recipes/update.rb

```diff
@@ -24,3 +24,18 @@
 include_recipe 'aws-parallelcluster-config::directory_service'
 include_recipe 'aws-parallelcluster-slurm::update' if node['cluster']['scheduler'] == 'slurm'
 include_recipe 'aws-parallelcluster-scheduler-plugin::update' if node['cluster']['scheduler'] == 'plugin'
+
+# Update node package - useful for development purposes only
+if !node['cluster']['custom_node_package'].nil? && !node['cluster']['custom_node_package'].empty?
+
+  service "supervisord" do
+    action :stop
+  end
+
+  include_recipe 'aws-parallelcluster-install::parallelcluster_custom_node'
```

Comment:
isn't better to postpone the node installation before the slurm::update? This because during the slurm update the clustermgtd will be restarted

Reply:
That happens only if the queues are update, right? This PR instead is for testing the node logic, which is independent of the cluster configuration.